types/supercluster/index.d.ts

```diff
@@ -1,11 +1,13 @@
-// Type definitions for supercluster 2.3
+// Type definitions for supercluster 3.0
 // Project: https://github.com/mapbox/supercluster
 // Definitions by: Denis Carriere <https://github.com/DenisCarriere>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import * as GeoJSON from 'geojson';
 
-interface Options {
+export as namespace supercluster;
```

Comment:
This could be removed, doesn't seem to bring any value to the Typescript definition and possibly might overlap on some existing types.

![image](https://user-images.githubusercontent.com/550895/31285841-69f73140-aa8a-11e7-889d-1374b99b2ca6.png)